service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -131,6 +131,24 @@ void shouldReturnChildrenDetailsWhenPopulatedChildren() {
         assertThat(service.getChildrenDetails(listChildren)).containsOnly(expectedChild);
     }
 
+    @Test
+    void shouldReturnChildrenDetailsWhenPopulatedChildrenDoesNotHaveDob() {
+        List<Element<Child>> listChildren = wrapElements(Child.builder()
+            .party(ChildParty.builder()
+                .firstName("John")
+                .lastName("Smith")
+                .build())
+            .build());
+
+        DocmosisChild expectedChild = DocmosisChild.builder()
+            .name("John Smith")
+            .gender("Male")
```

Comment:
Haven't specified a gender in the built child above

Reply:
stupid me :) that explains test failure ;)done